docs/build/getting-started/flow-cli.md

```diff
@@ -134,8 +138,24 @@ To deploy the `Counter` contract to the emulator, run:
 flow project deploy
 ```
 
+You'll see something similar to:
+
+```zsh
+Deploying 1 contracts for accounts: test-account
+
+Counter -> 0x179b6b1cb6755e31 (a98c155fe7afc8eb2af5551748759b08a80a0ae85d1b09f92f1afc293c61ca98)
+
+🎉 All contracts deployed successfully
+```
+
 That's it! You've just deployed your first contract to the Flow Emulator.
 
+::warning
+
+You can't deploy the same contract to multiple accounts at the same time with the `deploy` command. If you've experimented with the above, you may need to manually edit the `"deployments"` property in `flow.json` to remove extra deployments.
```

Comment:
Is this true? You could put the contract to deploy in the flow.json under two accounts. Does that not work?